v1/src/main/java/com/google/cloud/teleport/templates/common/SpannerConverters.java

```diff
@@ -314,6 +357,42 @@ public void processElement(ProcessContext processContext) {
         processContext.output(read);
       }
 
+      /**
+       * Generate a column expression for use in a query based on user-supplied column and alias
+       * values.
+       *
+       * @param columnsToExport User-provided data in the format of "column1: alias1, column2,
+       *     column3: alias3".
+       * @return A prepared column expression suitable for use in a Data Query Language (DQL)
+       *     statement.
+       */
+      private String prepareColumnExpression(String columnsToExport) {
```

Comment:
This should be very easy to add a unit test for

Reply:
Thanks for the suggestion. Adding it.